flink-runtime/src/main/java/org/apache/flink/runtime/webmonitor/WebMonitorUtils.java

```diff
@@ -142,6 +149,33 @@ private static File resolveFileLocation(String logFilePath) {
 		}
 	}
 
+	/**
+	 * Get address of web monitor from configuration.
+	 *
+	 * @param configuration Configuration contains those for WebMonitor.
+	 * @param resolution Whether to try address resolution of the given hostname or not.
+	 *                   This allows to fail fast in case that the hostname cannot be resolved.
+	 * @return Address of WebMonitor.
+	 */
+	public static String getWebMonitorAddress(
+		Configuration configuration,
+		HighAvailabilityServicesUtils.AddressResolution resolution
+	) throws UnknownHostException {
```

Comment:
I think this formatting style differs from the rest of the code in this file.

Reply:
Re-formatted.